docs/src/test/java/docs/http/javadsl/HttpClientExampleDocTest.java

```diff
@@ -170,6 +186,28 @@ public void testSingleRequestExample() {
     //#single-request-example
   }
 
+  // compile only test
+  public void singleRequestInActorExample1() {
+    //#single-request-in-actor-example
+      class SingleRequestInActorExample extends AbstractActor {
+      final Http http = Http.get(context().system());
+      final ExecutionContextExecutor dispatcher = context().dispatcher();
+      final Materializer materializer = ActorMaterializer.create(context());
+
+       @Override
+       public Receive createReceive() {
+         return receiveBuilder()
+                 .match(String.class, url -> pipe(fetch(url), dispatcher).to(self()))
```

Comment:
Does this line fit in the code listing on the doc page, or would make the user scroll?

Reply:
Fits for me.

Reply:
Fits for me too.